libc/newhdrgen/tests/expected_output/test_header.h

```diff
@@ -40,10 +40,6 @@ _Float16 func_c(int, float) __NOEXCEPT;
 _Float16 func_d(int, float) __NOEXCEPT;
 #endif // LIBC_TYPES_HAS_FLOAT16
 
-#ifdef LIBC_TYPES_HAS_FLOAT16_AND_FLOAT128
```

Comment:
would it make more sense to leave this function here and add a new function for the entrypoint testing?

Reply:
fixed, let me if I'm good to merge